packages/auth/src/commands/auth/2fa/disable.ts

```diff
@@ -1,26 +1,17 @@
 import {Command} from '@heroku-cli/command'
-import * as Heroku from '@heroku-cli/schema'
 import cli from 'cli-ux'
 
 export default class Auth2faGenerate extends Command {
   static description = 'disables 2fa on account'
 
+  static example = '$ heroku auth:2fa:disable'
+
   static aliases = [
     'twofactor:disable',
     '2fa:disable',
   ]
 
-  static example = `$ heroku auth:2fa:disable
-Disabling 2fa on me@example.com... done`
-
   async run() {
-    cli.action.start('Disabling 2fa')
-    const headers = {Accept: 'application/vnd.heroku+json; version=3.with_vaas_info'}
-    const {body: account} = await this.heroku.get<Heroku.Account>('/account', {headers})
-    cli.action.start(`Disabling 2fa on ${account.email}`)
-    if (account.vaas_enrolled) this.error('Cannot disable 2fa via CLI\nPlease visit your Account page on the Heroku Dashboard to manage 2fa: https://dashboard.heroku.com/account')
-    if (!account.two_factor_authentication) this.error('2fa is already disabled')
-    const password = await cli.prompt('Password', {type: 'hide'})
-    await this.heroku.patch('/account', {body: {password, two_factor_authentication: false}})
+    cli.warn('DEPRECATION WARNING: this command has been removed, in favor of disabling MFA in your Account Settings in a browser.')
```

Comment:
since we removed the code, this doesn't feel like a deprecation to me, it feels more like an error. If a customer is scripting or running this command in a sequence, I think we'd want to have the script stop execution with a non-zero exit status (indicating failure). It would be great to include the Dashboard URL also so developers can click and know where to go:


```suggestion
    cli.error('this command has been removed, in favor of disabling MFA in your Account Settings in a browser: https://dashboard.heroku.com/account')
    cli.exit(1)
```

Reply:
great suggestion, thanks!